apps/wakunode2/external_config.nim

```diff
@@ -414,6 +414,14 @@ type
         defaultValue: false
         name: "rest-private" }: bool
 
+      restAllowOrigin* {.
+        desc: "Allow cross-origin requests from the specified origin." &
+              "Argument may be repeated." &
+              "Wildcards: * or ? allowed." &
+              "Ex.: \"localhost:*\" or \"127.0.0.1:8080\"",
+        defaultValue: @[""]
```

Comment:
I think is better to just create an empty seq in this case because in another place we are checking `restAllowOrigin.len > 0`:
```suggestion
        defaultValue: newSeq[string]()
```